models/review.model.js

```diff
@@ -1,9 +1,23 @@
 const db = require('../db/connection.js')
 
 
-exports.fetchReviews = (id) => {
+exports.fetchReviews = (id, query) => {
     if(typeof id === "number"){
-        return db.query("SELECT * FROM reviews WHERE review_id = $1 ",[id]).then(({rows}) => {
+        if(query === "comment"){
```

Comment:
This needs changing so that its no longer query based, and uses JOIN to join the tables together, in order to get the comment_count